ui/components/search/ReservationUnitCard.tsx

```diff
@@ -226,23 +220,38 @@ const ReservationUnitCard = ({
             ) : null}
           </Props>
           <Actions>
-            <div style={{ flexGrow: 1 }} />
             {containsReservationUnit(reservationUnit) ? (
               <MediumButton
-                iconLeft={<IconCheck />}
+                iconRight={<IconCheck />}
                 onClick={() => removeReservationUnit(reservationUnit)}
+                data-testid="reservation-unit-card__button--select"
               >
                 {t("common:removeReservationUnit")}
               </MediumButton>
             ) : (
-              <MediumButton
-                iconLeft={<IconPlus />}
+              <BlackButton
+                iconRight={<IconPlus />}
                 onClick={() => selectReservationUnit(reservationUnit)}
                 variant="secondary"
+                data-testid="reservation-unit-card__button--select"
               >
                 {t("common:selectReservationUnit")}
-              </MediumButton>
-            )}
+              </BlackButton>
+            )}{" "}
+            <Link
+              href={link}
+              target="_blank"
+              rel="noopener noreferrer"
+              style={{ display: "contents" }}
+            >
+              <BlackButton
```

Comment:
Button inside a Link is suspicious.
Does <BlackButton as="span"... work for HDS components?

Reply:
Handled link in button.